File: Models/OreLocationElement.cs
#nullable disable

using System.Text.Json.Serialization;

namespace WebPage.Models
{
    public class OreLocationElement
    {
        public int Id { get; set; }
        [JsonPropertyName("rarity")]
        public OreRarity Rarity { get; set; }
        [JsonPropertyName("material")]
        public Material Material { get; set; }
        public OreLocation RootOreLocation { get; set; } = new OreLocation();
    }

}

File: ClientApp/src/app/api/models/ore-location-element.ts
/* tslint:disable */
/* eslint-disable */
import { Material } from '../models/material';
import { OreLocation } from '../models/ore-location';
import { OreRarity } from '../models/ore-rarity';
export interface OreLocationElement {
  id?: number;
  material?: Material;
  rarity?: OreRarity;
  rootOreLocation?: OreLocation;
}
